apps/src/templates/HeaderBanner.jsx

```diff
@@ -119,16 +119,16 @@ const styles = {
     maxWidth: styleConstants['content-width']
   },
   bannerHeading: {
-    fontFamily: '"Gotham 7r", sans-serif',
+    fontFamily: '"Barlow Semi Condensed Semibold", sans-serif',
     color: color.white,
-    fontSize: 32,
-    lineHeight: '40px'
+    fontSize: 48,
+    marginBottom: 40
```

Comment:
Can you remove this `margin-bottom` or make it `0` so there's not so much space between the h1's and subheads in these banners? 

It should look more like this:
<img width="1384" alt="Screenshot 2023-01-18 at 8 46 11 AM" src="https://user-images.githubusercontent.com/9256643/213242874-dd49a922-c9cd-42c9-b992-ff2b6f1b37b2.png">

Reply:
And the `line-height` looks like it'll be added once you merge in this change: https://github.com/code-dot-org/code-dot-org/pull/49831#discussion_r1073768191

Reply:
Yeah, this one should be fixed by merging https://github.com/code-dot-org/code-dot-org/pull/49831#discussion_r1073768191